packages/atomic/src/pages/v1.html

```diff
@@ -143,10 +143,11 @@
         display: none;
       }
 
-      atomic-search-box {
+      atomic-search-box-v1 {
         grid-area: search;
         margin: var(--spacing) 0;
         max-width: var(--search-box-width);
+        height: 48px;
```

Comment:
I would prefer to use `rem` here.
```suggestion
        height: 3rem;
```